mtcpclient/reporting.go

```diff
@@ -38,3 +38,45 @@ func StartBackgroundReporting(numberConnections int, rinterval int) (chan tcpcli
 
 	return connStatusCh, connStatusTracker
 }
+
+func FinalMetricsReport(connectionDescriptions []tcpclient.Connection) string {
+	mr := calculateMetricsReport(connectionDescriptions)
+	return "Time to establish TCP connections min/avg/max/stdDev = " +
+		mr.minToEstablished.String() + "/" +
+		mr.avgToEstablished.String() + "/" +
+		mr.maxToEstablished.String() + "/" +
+		mr.stdDevToEstablished.String()
+}
+
+type metricsReport struct {
```

Comment:
Probably we can find a better name. This "metric-group stats" is being consumed by the report, but its not a report by itself. Maybe metricsCollectionStats?

Reply:
I like it